Makefile

```diff
@@ -303,7 +340,14 @@ guard-%:
 prd/robots.txt: scripts/robots.mako-dot-txt .build-artefacts/last-deploy-target
 	mkdir -p $(dir $@)
 	${PYTHON_CMD} ${MAKO_CMD} \
-	    --var "deploy_target=$(DEPLOY_TARGET)" $< > $@
+	    --var "version=$(VERSION)" \
+	    --var "deploy_target=dev" $< > $@
+
+prd/robots_prod.txt: scripts/robots.mako-dot-txt .build-artefacts/last-deploy-target
+	mkdir -p $(dir $@)
+	${PYTHON_CMD} ${MAKO_CMD} \
+	    --var "version=$(VERSION)" \
+	    --var "deploy_target=prod" $< > $@
```

Comment:
For the robots target, we could remove the last-deploy-target pre-requisite (as its hard-coded now). And we should add the last-version prerequiste. The mako itselfl contains the version.